### PR TITLE
Add favorite functionality and remove dummy card

### PR DESCRIPTION
**Pull request by colefiscus**
#### What does  this PR do?

- Add favorite functionality, including altering star appearance. A user can favorite or unfavorite and star will change appearance. Removed placeholder card. Added small syntax edits.

#### Where should the reviewer start?

- Pay attention to lines 82 and 40 to see added function and change to showIdeas().

#### How should this be manually tested?

- Add idea cards and ensure that star will change appearance with each click.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,22 +26,7 @@ <h2>Filter Starred Ideas</h2>
         </div>
       </div>
     </section>
-    <section class="idea-grid">
```
Comment (DrewBradley) [inline index.html:29]: Thank you for removing this. The HTML looks much cleaner now!

File: src/main.js [hunk 2]
```diff
@@ -6,45 +6,47 @@ var bodyInput = document.querySelector(".body-input");
 var saveButton = document.querySelector(".save-button");
 var searchInput = document.querySelector(".search-input");
 var searchButton = document.querySelector(".search-button");
-var starButton = document.querySelector(".like");
-var deleteButton = document.querySelector(".delete");
-var commentButton = document.querySelector(".comment-button");
-var ideaGrid = document.querySelector(".idea-grid")
+var ideaGrid = document.querySelector(".idea-grid");
 
 var savedIdeas = [];
```
Comment (JeffKersting) [inline src/main.js:11]: Create local storage for savedIdeas array.

File: src/main.js [hunk 3]
```diff
@@ -6,45 +6,47 @@ var bodyInput = document.querySelector(".body-input");
 var saveButton = document.querySelector(".save-button");
 var searchInput = document.querySelector(".search-input");
 var searchButton = document.querySelector(".search-button");
-var starButton = document.querySelector(".like");
```
Comment (DrewBradley) [inline src/main.js:9]: Good catch Cole, we didn't need all these extra query selectors!

File: src/main.js [hunk 4]
```diff
@@ -55,24 +57,34 @@ function showIdeas() {
...
-  showIdeas();
-  }
+function deleteIdea(target) {
+  var id = parseInt(target.parentNode.parentNode.id);
+    for (var i = 0; i < savedIdeas.length; i++) {
+      if (savedIdeas[i].id === id) {
+        savedIdeas.splice(i, 1);
+      };
+    };
+    showIdeas();
```
Comment (JeffKersting) [inline src/main.js:79]: Fire Idea class method to remove from local storage.

File: src/main.js [hunk 5]
```diff
@@ -6,45 +6,47 @@ var bodyInput = document.querySelector(".body-input");
...
+  };
+};
 
 function showIdeas() {
   ideaGrid.innerHTML = "";
-  for (var i = 0; i < savedIdeas.length; i++){
+  for (var i = 0; i < savedIdeas.length; i++) {
+    if (savedIdeas[i].star === false) {
+      var star = "assets/star.svg";
+    } else if (savedIdeas[i].star === true) {
```
Comment (DrewBradley) [inline src/main.js:43]: Amazing! Separate that DOM and data model. Beautifully done Cole!

**Review by DrewBradley — APPROVED** [review]

**Review by JeffKersting — APPROVED** [review]

**Pull request merged**